src/test/regress/expected/propagate_statistics.out

```diff
@@ -51,6 +51,11 @@ SELECT create_distributed_table ('test_stats3','a');
 
 (1 row)
 
+-- test dropping statistics
+CREATE STATISTICS s3 ON a,b FROM test_stats3;
+CREATE STATISTICS sc2.s4 ON a,b FROM test_stats3;
+-- s6 doesn't exist
+DROP STATISTICS IF EXISTS s3, sc2.s4, s6;
```

Comment:
maybe include some mixture of statistics on distributed, reference and local tables.